### ISSUE TITLE
Add Redis resource utilization benchmarks

### ISSUE THREAD

**Issue by skeggse** [bee-queue/bee-queue#67]
We updated the benchmarks to show throughput for job processing across different job queues, but didn't instrument or compare their Redis utilization. We should do that.

**Comment by manast** [comment]
I recommend you do that after this PR is merged: https://github.com/OptimalBits/bull/pull/641

**Comment by skeggse** [comment]
Some related results: #69. Unfortunately not a complete Redis util benchmark.

**Comment by bradvogel** [comment]
Thanks for adding. So it looks like based on the raw results, Bee Queue is still roughly 2x faster than Bull 3.0rc5. Cc @manast

**Comment by manast** [comment]
@bradvogel hmm, more like 1.5x I would say. I am quite happy with the results so far. The concurrency issue is gone which was the most important issue in 3.0.

**Comment by manast** [comment]
Btw. something to consider. Bull has many more .lua files than bq to read and compile when it starts. This time is also computed in the test, could add 0.5 to 1 second to the total benchmark time.

**Comment by bradvogel** [comment]
Youre right - about 1.5x. Not sure about the startup time, but that could have also influenced the tests. Looks like Bull 3rc5 is getting close, and has just fine performance for 99% of queue use cases.
